tests/python_client/testcases/test_insert.py

```diff
@@ -1538,7 +1538,9 @@ def test_upsert_multiple_times(self):
         for i in range(10):
             data = cf.gen_default_data_for_upsert(upsert_nb, start=i*500)[0]
             collection_w.upsert(data)
-        assert collection_w.num_entities == upsert_nb*10 + ct.default_nb
+        # check the result
+        res = collection_w.query(expr="", output_fields=["count(*)"])[0]
+        assert res[0]["count(*)"] == 5500
```

Comment:
do not hard code 5500 here, or it would fail if you updated the value of nb some day

Reply:
> do not hard code 5500 here, or it would fail if you updated the value of nb some day

Thanks for your advice! I'll calculate the result the modify the assert.